src/autocomplete/src/Autocomplete.js

```diff
@@ -237,7 +238,9 @@ export default class Autocomplete extends PureComponent {
                   },
                   inputValue,
                   selectedItem,
-                  highlightedIndex,
+                  highlightedIndex: this.props.items.includes(selectedItem)
+                    ? this.props.items.indexOf(selectedItem)
+                    : 0,
                   selectItemAtIndex,
                   ...restDownshiftProps
```

Comment:
By removing it from the destructure at the top it is being passed in `...restDownshiftProps`. Your code change for `children` here will be ignored because it is being overwritten by the value later.

Reply:
Oops, you're right! 
I'll add highlightedIndex to the destructure